managers/engineering_manager.md

```diff
@@ -19,46 +19,63 @@ It's the responsibility of the EM to ensure all of the following for their teams
 
 #### Planning
 
-* Work with PM/ED/TL/Designer to create key results for teams
+* Work with cross-team representatives to create key results for teams, including:
+   * Product managers (PM)
+   * Engineering directors (ED) 
+   * Team leads (TL)
+   * Designers
 
 #### Process
 
 * Evolve Agile approach to fit team and project needs.
 * Foster continuous improvement mindset amongst team
 * Challenge team to identify bottlenecks & ways of speeding up delivery
+* Ensure engineers are focused on the sprint & stories at hand
+* Ensure engineers understand how the work they are doing impacts objects and key results (OKRs) & wider company goals
 
 #### Meetings
 
-* Guide effectiveness of team meetings.
-* Ensure agendas/purpose are clear & actions captured.
+* Guide effectiveness of team meetings
+* Ensure agendas/purpose are clear & actions captured
 
 #### Determine team/team makeup & hiring needs
 
 * Ensure team has the people and skill sets needed to achieve their goals
-* Recruit for the team
-   * Source engineers
+* Assist in recruitment for the team:
+   * Source engineers and facilitate their introduction to the recruitment pipeline
    * Motivate engineers to speak, do presentations, write blog posts, and contribute to open source projects (and do so oneself!)
-   * Update interview loops & training engineers to use them for hiring
-   * Work with recruiting to ensure the loops run smoothly
-   * Make the final hiring decision
```

Comment:
@bashhack I see this is removed. Has this been passed on to another role's responsibilities or is this now reflected as a group desicion?

Reply:
These lines really came from the underlying Meetup role content - and while there was much there we could keep - at least at MT (or at least in my personal experience) the `final hiring decision` and `working with recruiters` occurs at the level of the directors rather than engineering managers.

That said, I did try to capture the involvement of managers in the process with these new lines:

```
Update interview processes & train engineers to use them for hiring
Work with hiring team to ensure interview processes reflect team values and needs
```

If your experience is driving you toward a different edit - let's not be shy about revising to better match the reality of our team!